### ISSUE TITLE
Question about code, half seconds

### ISSUE THREAD

**Issue by yene** [jishi/node-sonos-discovery#98]
https://github.com/jishi/node-sonos-discovery/blob/master/lib/Subscriber.js#L56

Why are you using half seconds for the subscribe timeout?

**Comment by jishi** [comment]
It's because I need to specify a subscription timeout when I subscribe for events, but I want to re-subscribe before that timeout actually happens. I settled on half the time (meaning, I ask for a 10 minute subscription, but resubscribe after 5 minutes). Previously I used 0.9 of the subscription timeout (9 minutes) but I got the feeling that it sometimes stopped before resubscribing. And it doesn't seem to be an issue with resubscribing early, so I settled for half.

The code you marked is the delay until resubscribing, the timeout sent to the Sonos player is this:

https://github.com/jishi/node-sonos-discovery/blob/master/lib/Subscriber.js#L39
